test/odbc/psqlodbc/psqlodbc_tests_common.cpp

```diff
@@ -556,7 +556,8 @@ void testComparisonOperators(ServerType serverType, const string &tableName, con
 
   for (int i = 0; i < NUM_OF_DATA; i++) {
     odbcHandler.CloseStmt();
-    odbcHandler.ExecQuery(SelectStatement(tableName, operationsQuery, vector<string>{}, col1Name + "=\'" + col1Data[i] + "\'"));
+    const string WHERE_STATEMENT = explicitCast ? col1Name + " OPERATOR(sys.=) " + col1Data[i] + "" : col1Name + "=\'" + col1Data[i] + "\'";
```

Comment:
What about in cases where we need quotation marks around the value we're comparing in the WHERE clause?

I.e. `WHERE data OPERATOR(sys.=) 1` vs
`WHERE data OPERATOR(sys.=) '1900-01-01'`